airflow/providers/google/ads/operators/ads.py

```diff
@@ -89,11 +92,19 @@ def __init__(
         self.google_ads_conn_id = google_ads_conn_id
         self.gzip = gzip
         self.impersonation_chain = impersonation_chain
+        self.api_version = api_version
 
     def execute(self, context: dict) -> str:
         uri = f"gs://{self.bucket}/{self.object_name}"
+        kwargs = {}
+        if self.api_version:
+            kwargs.update(api_version=self.api_version)
 
-        ads_hook = GoogleAdsHook(gcp_conn_id=self.gcp_conn_id, google_ads_conn_id=self.google_ads_conn_id)
+        ads_hook = GoogleAdsHook(
+            gcp_conn_id=self.gcp_conn_id,
+            google_ads_conn_id=self.google_ads_conn_id,
+            **kwargs,
+        )
```

Comment:
Wouldn't it be simpler to do:
```
class GoogleAdsHook:
  default_version = "v5"
  
  def __init__(self, version):
    self.version = version or self.default_version
```

In that way we can simply pass `None` values in operators. WDYT?

Reply:
@dstandish Suggested that way on this review comment. https://github.com/apache/airflow/pull/15266#discussion_r619697979

What do you think @dstandish?

Reply:
i think either way is ok but i like @turbaszek's suggestion the best

it's all tradeoffs, but i think @turbaszek approach minimizes compleixity and noise overall, because as he points out then you don't need the kwargs logic.